src/containers/LoginPage/index.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import LoginForm from '../../components/LoginForm';
+import './LoginPage.scss';
+
+const LoginPage = () => (
+  <div className="container-fluid">
+    <h1> This is the Login Page </h1>
+    <p>Our company is great, here is more info about us</p>
+    <div className="form-container">
+      <LoginForm />
```

Comment:
We could probably get away with only including the login form on the page, horizontally centered using Bootstrap (i.e., disregard the messaging on the left) since it's relatively unclear what messaging will go there for now.